jsonpb/jsonpb_test.go

```diff
@@ -752,6 +754,13 @@ var unmarshalingTests = []struct {
 	{"UInt32Value", Unmarshaler{}, `{"u32":4}`, &pb.KnownTypes{U32: &wpb.UInt32Value{Value: 4}}},
 	{"BoolValue", Unmarshaler{}, `{"bool":true}`, &pb.KnownTypes{Bool: &wpb.BoolValue{Value: true}}},
 	{"StringValue", Unmarshaler{}, `{"str":"plush"}`, &pb.KnownTypes{Str: &wpb.StringValue{Value: "plush"}}},
+	{"StructValue containing StringValue's", Unmarshaler{}, "{\"escaped\": \"a\\/b\", \"unicode\": \"\\u00004E16\\u0000754C\"}",
+		&stpb.Struct{
+			Fields: map[string]*stpb.Value{
+				"escaped": {Kind: &stpb.Value_StringValue{"a/b"}},
```

Comment:
This testcase exposes issue #605.

Note that the other testcases that I added here are somehow working even before adding the fix in.